src/client/src/apps/MainRoute/widgets/spotTile/components/PriceControls/PriceControls.tsx

```diff
@@ -71,6 +71,7 @@ const PriceControls: React.FC<Props> = ({
     isRfqStateNone || isRfqStateReceived || isRfqStateExpired || isTradeExecutionInFlight
   const priceMovement = hasPrice ? priceData.priceMovementType : 'none'
   const spreadValue = hasPrice ? spread.formattedValue : '-'
+  const showPriceMovement = isRfqStateNone && !isTradeExecutionInFlight ? true : false
```

Comment:
maybe you don't need the Conditional Operator `isRfqStateNone && !isTradeExecutionInFlight` will return true or false anyway